package.json

```diff
@@ -30,7 +30,7 @@
     "core-js": "^2.5.7",
     "express": "^4.16.4",
     "jquery": "^3.3.1",
-    "less": "^2.7.3",
+    "less": "^3.9.0",
```

Comment:
I'm guessing these packages were updated unintentionally which makes me nervous. Could you try installing using package-lock instead.

Reply:
I'm pretty sure I added all of these by mistake and then reverted the changes in the final commit. So I don't think any of them are actually what is up for review. At least I don't mean any of them to be.